Incorrect error message when over-using empty string

# Bug report

<!--
Hi, thanks for submitting a bug. We appreciate that.

But, we will need some information about what's wrong to help you.
-->

## What's wrong

Right now:

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: 

## How is that should be

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: ''

I am using `0.10`

Hey!
I wrote a new test. And test falls.
```
 assert "Z226 Found s... over-use: ''" == 'Z226 Found st...nt over-use: '
E         - Z226 Found string constant over-use: ''
E         ?                                      --
E         + Z226 Found string constant over-use:
```
I add the following rule to the code
```
 def _post_visit(self) -> None:
        for string, usage_count in self._string_constants.items():
            if not len(string):
                string = "''"
            if usage_count > self.options.max_string_usages:
                self.add_violation(OverusedStringViolation(text=string))
```
Then the previous test falls.
How to catch an empty line for a visitor?

--- a/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
+++ b/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
@@ -41,7 +41,7 @@ def test_string_overuse_settings(
 ])
 @pytest.mark.parametrize('string_value', [
     '"same-string"',
-    "''"
+    "''",
 ])
 def test_string_overuse(
     assert_errors,

--- a/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
+++ b/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+import pytest
+
 from wemake_python_styleguide.violations.complexity import (
     OverusedStringViolation,
 )
@@ -12,16 +14,20 @@ third[{0}]
 'new' + {0}
 """
 
-string_values = ['"same-string"', "''"]
 
-
+@pytest.mark.parametrize('strings', [
+    string_actions,
+])
+@pytest.mark.parametrize('string_value', ['"same_string"'])
 def test_string_overuse_settings(
     assert_errors,
     parse_ast_tree,
     options,
+    strings,
+    string_value,
 ):
     """Ensures that settings for string over-use work."""
-    tree = parse_ast_tree(string_actions.format('"same-string"'))
+    tree = parse_ast_tree(strings.format(string_value))
 
     option_values = options(max_string_usages=4)
     visitor = WrongStringVisitor(option_values, tree=tree)
@@ -30,18 +36,26 @@ def test_string_overuse_settings(
     assert_errors(visitor, [])
 
 
+@pytest.mark.parametrize('strings', [
+    string_actions,
+])
+@pytest.mark.parametrize('string_value', [
+    '"same-string"',
+    "''"
+])
 def test_string_overuse(
     assert_errors,
     assert_error_text,
     parse_ast_tree,
     default_options,
+    strings,
+    string_value,
 ):
     """Ensures that over-used strings raise violations."""
-    for string_val in string_values:
-        tree = parse_ast_tree(string_actions.format(string_val))
+    tree = parse_ast_tree(strings.format(string_value))
 
-        visitor = WrongStringVisitor(default_options, tree=tree)
-        visitor.run()
+    visitor = WrongStringVisitor(default_options, tree=tree)
+    visitor.run()
 
-        assert_errors(visitor, [OverusedStringViolation])
-        assert_error_text(visitor, string_val.replace('"', ''))
+    assert_errors(visitor, [OverusedStringViolation])
+    assert_error_text(visitor, string_value.replace('"', ''))

--- a/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
+++ b/tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py
@@ -12,7 +12,7 @@ third[{0}]
 'new' + {0}
 """
 
-string_value = '"same-string"'
+string_values = ['"same-string"', "''"]
 
 
 def test_string_overuse_settings(
@@ -37,10 +37,11 @@ def test_string_overuse(
     default_options,
 ):
     """Ensures that over-used strings raise violations."""
-    tree = parse_ast_tree(string_actions.format(string_value))
+    for string_val in string_values:
+        tree = parse_ast_tree(string_actions.format(string_val))
 
-    visitor = WrongStringVisitor(default_options, tree=tree)
-    visitor.run()
+        visitor = WrongStringVisitor(default_options, tree=tree)
+        visitor.run()
 
-    assert_errors(visitor, [OverusedStringViolation])
-    assert_error_text(visitor, string_value.replace('"', ''))
+        assert_errors(visitor, [OverusedStringViolation])
+        assert_error_text(visitor, string_val.replace('"', ''))

--- a/wemake_python_styleguide/visitors/ast/builtins.py
+++ b/wemake_python_styleguide/visitors/ast/builtins.py
@@ -45,7 +45,7 @@ class WrongStringVisitor(BaseNodeVisitor):
     def _post_visit(self) -> None:
         for string, usage_count in self._string_constants.items():
             if usage_count > self.options.max_string_usages:
-                self.add_violation(OverusedStringViolation(text=string))
+                self.add_violation(OverusedStringViolation(text=string or "''"))
 
     def visit_Str(self, node: ast.Str) -> None:
         """

